### DISCUSSION TITLE
Will this support routify?

### DISCUSSION THREAD

**Discussion by paradis-A** [pixelmund/svemix#45] [Q&A] [answered]
Waiting for Sveltekit goes v1, Im using routify at the moment. By looking at the docs its like all my problem has been solved. Still have loads of stuff to do, I cant just jump from another boat at the moment. Hopefully someone tried it then might able to share it here.

**Comment by pixelmund** [comment, marked as answer]
I don't think that svemix will support routify, svemix is tightly coupled to SvelteKit.
